Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:18.04
+
+ENV WORKDIR=/app
+
+WORKDIR ${WORKDIR}
+
+# install Python3.8.2
+#https://linuxize.com/post/how-to-install-python-3-8-on-debian-10/
+RUN apt-get update -y \
+    && apt-get install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libsqlite3-dev libreadline-dev libffi-dev curl liblzma-dev libbz2-dev
+RUN curl -O https://www.python.org/ftp/python/3.8.2/Python-3.8.2.tar.xz \
+    && tar -xf Python-3.8.2.tar.xz
+RUN cd Python-3.8.2 \
+    && ./configure --enable-optimizations \
+    && make -j `nproc` \
+    && make altinstall \
+    && python3.8 --version \
+    && cd ..
+
```

Comment:
Why not just use the `python:3.8.2` Docker image?

Reply:
I don't know what that image is based on, I feel comfortable using a Debian based distro.
Also I'm not sure that it includes the python dependencies that we need for Pandas.
Do you want to check it?

Reply:
It’s based upon Debian. You can install other dependencies - it just saves you the entire Python 3.8.2 compilation step and quite frankly it seems smarter to use the recommended Python Docker image which will be built to standards than to roll your own.

Reply:
done